### PR TITLE
Ccalc 2991

### PR DESCRIPTION
**Pull request by zatevad**

### CODE REVIEW

File: project/MicroService.scala [hunk 1]
```diff
@@ -20,7 +20,7 @@ trait MicroService {
   lazy val scoverageSettings = {
     import scoverage._
     Seq(
-      ScoverageKeys.coverageExcludedPackages :=  "<empty>;Reverse.*;uk.gov.hmrc.BuildInfo.*;app.Routes.*;prod.*;uk.gov.hmrc.childcarecalculatorfrontend.config.*;uk.gov.hmrc.childcarecalculatorfrontend.views.html.common.*",
+      ScoverageKeys.coverageExcludedPackages :=  "<empty>;Reverse.*;uk.gov.hmrc.BuildInfo.*;app.Routes.*;prod.*;uk.gov.hmrc.childcarecalculatorfrontend.config.*;uk.gov.hmrc.childcarecalculatorfrontend.views.html.common.*;uk.gov.hmrc.childcarecalculatorfrontend.utils.EnumUtils;uk.gov.hmrc.childcarecalculatorfrontend.models.*",
```
Comment (kapil-sanghvi) [inline project/MicroService.scala]: uk.gov.hmrc.childcarecalculatorfrontend.utils.EnumUtils - not sure about this one to be excluded

File: test/uk/gov/hmrc/childcarecalculatorfrontend/controllers/ChildAgedTwoControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        when(
+          sut.keystore.cacheEntryForSession[Boolean](anyString, anyBoolean)(any[HeaderCarrier], any[Format[Boolean]])
+        ).thenReturn(
+          Future.successful(Some(true))
+        )
+
+        val result = await(
+          sut.onSubmit(
+            request
+              .withFormUrlEncodedBody(form.data.toSeq: _*)
```
Comment (kapil-sanghvi) [inline test/uk/gov/hmrc/childcarecalculatorfrontend/controllers/ChildAgedTwoControllerSpec.scala]: we could write .withFormUrlEncodedBody(childAgedTwoKey -> "true") and avoid creating form objects, unless it is done for some specific reason

Comment (zatevad) [inline test/uk/gov/hmrc/childcarecalculatorfrontend/controllers/ChildAgedTwoControllerSpec.scala]: Using childAgedTwoKey -> "true" gives null pointer exceptions so I've left it as is

Comment (kapil-sanghvi) [inline test/uk/gov/hmrc/childcarecalculatorfrontend/controllers/ChildAgedTwoControllerSpec.scala]: val result = await(
          sut.onSubmit(
            request
              .withFormUrlEncodedBody(childAgedTwoKey -> "true")
              .withSession(validSession)
          )
        )

strange, it does work for me you can refer branch CCALC-3008 as well for ExpectChildcareCostsControllerSpec

**Pull request merged**